lua/wikis/commons/Widget/MainPage/LiquipediaApp.lua

```diff
@@ -0,0 +1,115 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/MainPage/LiquipediaApp
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Image = require('Module:Image')
+local Lua = require('Module:Lua')
+
+local Info = Lua.import('Module:Info', {loadData = true})
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Div = HtmlWidgets.Div
+local IconFa = Lua.import('Module:Widget/Image/Icon/Fontawesome')
+local WidgetUtil = Lua.import('Module:Widget/Util')
+
+local GREEN_CHECK_CIRCLE = IconFa{
+	iconName = 'checkcircle',
+	color = 'forest-green-non-text',
+	additionalClasses = { 'fa-lg' },
+	additionalCss = {
+		['margin-left'] = '-0.1em',
+		['margin-right'] = '0.3em'
+	}
+}
+
+---@class LiquipediaApp: Widget
+---@operator call(table): LiquipediaApp
+local LiquipediaApp = Class.new(Widget)
+
+---@param entries (string|Html|Widget|nil|(string|Html|Widget|nil)[])[][]
+---@return Widget
+local function buildDescriptionList(entries)
+	return HtmlWidgets.Dl{
+		children = Array.map(entries, function (entry)
+			return HtmlWidgets.Dd{
+				children = WidgetUtil.collect(unpack(entry))
+			}
+		end)
+	}
+end
+
+---@return Widget[]
+function LiquipediaApp:render()
+	return {
+		Div{
+			css = {
+				display = 'Flex',
+				['justify-content'] = 'left',
+				padding = '0px',
+			},
+			children = {
+				Div{
+					classes = { 'mobile-hide' },
+					children = {Image.display('Qr-code-app.svg', nil, { size = '132px', link = '' })}
+				},
+				Div{
+					css = { ['line-height'] = '2.2em' },
+					children = buildDescriptionList{
+						{
+							GREEN_CHECK_CIRCLE,
+							'Follow your favorite ',
+							Info.name,
+							' players and teams!'
+						},
```

Comment:
The green check seems likely to be repeated for all list items? Maybe make it part of the description list instead